include/ray.h

```diff
@@ -9,20 +9,28 @@ class Ray {
 private:
   Vertex origin_;
   Direction direction_;
+  float importance_;
```

Comment:
It doesn't drop for pure reflection and still needs to be implemented I think ? https://github.com/novalain/gi-ray/issues/86